docs_user/modules/proc_adopting-compute-services-to-the-data-plane.adoc

```diff
@@ -166,17 +162,30 @@ data:
 EOF
 ----
 +
+[NOTE]
 The secret `nova-cell<X>-compute-config` is auto-generated for each
```

Comment:
Maybe:
"The secret `nova-cell<X>-compute-config` auto-generates for each `cell<X>`."